src/serialization/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2016-2019, The Monero Project
+# Copyright (c) 2016-2020, The Monero Project
```

Comment:
Do you want to update just the copyright year in the file?

Reply:
Reverted.

Reply:
Seems not

Reply:
This time the change was definitely added to the commit list.